src/client/src/apps/SimpleLauncher/index.tsx

```diff
@@ -1,16 +1,74 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
```

Comment:
this is a big change too - instead of being simple entry into launcher app, it is now also creating service instances and passes that though the contexts